Document custom game modes from osu!(lazer) in `Game mode` (#12264)
ppy/osu-wiki: wiki/Game_mode/en.md

# Game mode

osu! has four built-in **game modes** with different mechanics, originally inspired by other rhythm games:

- ![][osu!] [osu!](#osu!)
- ![][osu!taiko] [osu!taiko](#osu!taiko)
- ![][osu!catch] [osu!catch](#osu!catch)
- ![][osu!mania] [osu!mania](#osu!mania)

> To switch between modes, click the `Mode` button at the bottom of the Song Selection screen for osu!(Stable), use the buttons on the top of the screen for osu!(Lazer), or use these key combinations:

Reviewer: ```suggestion
To switch between modes, click the `Mode` button at the bottom of the Song Selection screen for osu!(stable), use the buttons on the top of the screen for osu!(lazer), or use these key combinations:
```

Author: .

- `Ctrl` + `1` for osu!
- `Ctrl` + `2` for osu!taiko
- `Ctrl` + `3` for osu!catch
- `Ctrl` + `4` for osu!mania

> ![Mode selection interface](/wiki/shared/Interface_mode.png "Mode selection interface in osu!(Stable)")

Reviewer: ```suggestion
![Mode selection interface](/wiki/shared/Interface_mode.png "Mode selection interface in osu!(stable)")
```

Author: .

> ![Mode selection interface](/wiki/shared/Interface_mode_lazer.png "Mode selection interface in osu!(Lazer)")

Reviewer: ```suggestion
![Mode selection interface](/wiki/shared/Interface_mode_lazer.png "Mode selection interface in osu!(lazer)")
```

Author: done

![Gameplay of osu!mania](/wiki/shared/mania-gameplay.jpg "osu!mania gameplay")

> ## Custom Game modes

Reviewer: ```suggestion
## Custom game modes
```

Author: Done

> In osu!(Lazer), you can use custom game modes (also called Custom Rulesets) that introduce a new way to play the game using the Beatmap Library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.

Reviewer: ```suggestion
In osu!(lazer), you can use custom game modes (also called custom rulesets) that introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
```

Author: done

> In osu!(lazer), you can use custom game modes (also called custom rulesets) that introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.

Reviewer: Forgot that "you" should be avoided: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#perspective

> In osu!(lazer), it's possible to use custom game modes (also called custom rulesets) that introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.

Reviewer: ```suggestion
In osu!(lazer), it's possible to manually install custom game modes (also called custom rulesets) via `.dll`. They introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
```

> The full list of playable Custom Game modes can be found [here](https://github.com/ppy/osu/discussions/13096).

Reviewer: Using "here" for links is discouraged IIRC, but couldn't see a reference in the Article Styling Criteria.

```suggestion
The full list of playable custom game modes can be found in [this GitHub discussion](https://github.com/ppy/osu/discussions/13096).
```

Author: Ahh, didn't know. Will change right now

Author: done
